src/v3/src/transformer/layout/googleAuthenticator/transformGoogleAuthenticatorEnroll.test.ts

```diff
@@ -98,6 +104,8 @@ describe('Google Authenticator Enroll Transformer Tests', () => {
 
   it('should add Stepper layout to UI Schema elements '
     + 'when GA Enroll params exists in Idx response', () => {
+    expect.hasAssertions();
```

Comment:
eslint isn't happy that the assertions are all within the `validateForm` helper function so this helps ensure we do actually make assertions and also makes the linter happy.